apps/crn-frontend/src/welcome/Welcome.tsx

```diff
@@ -50,7 +50,7 @@ const Welcome: React.FC<Record<string, never>> = () => {
       allowSignup
       onClick={() =>
         // Effect should populate this ref before a click can occur
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I couldn't find in the changelog but I believe this `@typescript-eslint/no-non-null-assertion` and `@typescript-eslint/no-empty-function` was changed in one of the versions in between the one we had and the last one because now we get this `Unused eslint-disable directive (no problems were reported from '@typescript-eslint/no-non-null-assertion')` and ` Unused eslint-disable directive (no problems were reported from '@typescript-eslint/no-empty-function')` in several files


<img width="1009" alt="Screenshot 2024-01-12 at 07 58 12" src="https://github.com/yldio/asap-hub/assets/16595804/78bcafb8-4296-4a15-a690-da239730229d">